stats.js

```diff
@@ -81,8 +81,8 @@ function stats(opts) {
             stats: stats
           })
 
-          previousCpu = cpuSum
-          previousSystem = sysSum
+          previousCpu = stats.cpu_stats.cpu_usage.total_usage
+          previousSystem = stats.cpu_stats.system_cpu_usage
```

Comment:
I've started to use a statsinterval>1 (e.g. 10 - to reduce load on the monitoring agent) and got always 0.0 as value for cpu_percent for some 'sleeping' containers. Then I debugged and discovered that previousCpu was higher than cpuSum and the difference got a negative value. There is a if condition that returns then 0.0 when negative - explaining why I got always 0.0. cpu percent. And I wondered how this could happen. After making this change my cpu value looked correct like 0.38% - the same value I had with statsinterval=1.